web-console/knikubevirt/Affinity/affinity.md

```diff
@@ -0,0 +1,61 @@
+# Affinity
+
+### About Affinity
+Affinity allows the user to create a relationship between two resources when scheduling them on the cluster.
+There are two high level relationship types:
+Affinity - attaching a group of resources together, making them run together in the same environment.
+Anti-affinity - creating a rejection relationship between a group of resources, thus avoiding scheduling any of them in the same environment.
+
+Besides defining if we want to attach or reject resources to one another, we can choose to "force" this rule we are creating.
+"Forcing" a rule means that the conditions of this rule must met, even if it means that some resources just won't be scheduled at all.
+
+### a few techincal words
+Affinity rules are Pod Properties, written in the pod deployment file. When a pod is scheduled, the deployment file states that there is an affinity rule condition which needs to be met.
+Applying an affinity rule to a resources is done via labels. Objects in OpenShift are ephemeral, and the way to target the same object which keeps crashing and regenerating is by its label.
+
+
+## Affinity rules - list page
+
+![Affinity list](img/D2-0-0.jpg)
```

Comment:
nit pic but should we change this to "Affinity Rules" for consistency? I know we are moving to sentence case anyway so the alternative could be changing all the other nav items to sentence case. @beanh66 WDYT? Any idea when the console will move to sentence case? The "Add Affinity rule" button should be sentence case too.

Reply:
For now, I've changed the button to title case but left the page header as sentence case

Reply:
I think the nav should be title case